libs/island-ui

```diff
@@ -0,0 +1 @@
+../submodules/island.is/libs/island-ui
```

Comment:
Do you need to symlink these into the libs folder for everything to work? Is it not enough to map the import paths in tsconfig straight into `submodules/island.is`.

It muddles the boundary between DMR code and island.is code, and might be confusing if devs try to edit these libs directly.

Reply:
I agree, I will take this approach instead.